### ISSUE TITLE
Installing a livepatch a second time prints an error message

### ISSUE THREAD

**Issue by scmschmidt** [SUSE/libpulp#176]
```
...
systemd (pid: 1): FAILED /usr/lib64/openssl-1_1-livepatches/0.1/[libcrypto_1.1.1l-150400.5.14.0_livepatch1.so](http://libcrypto_1.1.1l-150400.5.14.0_livepatch1.so/): Patch already applied
...
```

Even with this technically being correct, a 'FAILED' is not ideal. Maybe using 'SKIPPED' as well or introducing a new flag (e.g NOT NEEDED)?

**Comment by giulianobelinassi** [comment]
In which version this was tested? This 'error' is already marked as skippable: https://github.com/SUSE/libpulp/blob/0f32ab47a6879d782811b3b0bf19c288d4ade6d8/tools/trigger.c#LL331C2-L331C2

**Comment by giulianobelinassi** [comment]
This was fixed in libpulp 0.2.9. The current libpulp in SLE-15-SP4 is 0.2.8
